src/deion/worker/core/player/value.js

```diff
@@ -58,37 +66,51 @@ const value = (
 	const slope = 1.531;
 	const intercept = 31.693;
 
-	// 1. Account for stats (and current ratings if not enough stats)
+	// 1. OVERALL RATINGS + PER + CONSISTENCY
+	// Account for stats (and current ratings if not enough stats)
 	const ps = p.stats.filter(playerStats => !playerStats.playoffs);
 	let current = pr.ovr; // No stats at all? Just look at ratings more, then.
 	if (process.env.SPORT === "basketball" && ps.length > 0) {
 		const ps1 = ps[ps.length - 1]; // Most recent stats
-
-		if (ps.length === 1 || ps[0].min >= 2000) {
+		if (ps.length === 1) {
 			// Only one year of stats
-			current = intercept + slope * ps1.per;
-			if (ps1.min < 2000) {
-				current = (current * ps1.min) / 2000 + pr.ovr * (1 - ps1.min / 2000);
+			if (ps1.min > 14 * ps1.gp) {
+				//high mpg
+				current = 0.6 * current + 0.4 * ps1.per;
+			} else {
+				//low mpg
+				current = 0.875 * current + 0.125 * ps1.per;
```

Comment:
I'm concerned this will do weird things for players with very low minutes, like imagine a scrub who plays 1 minute and hits a shot, now his PER is 200 and his value shoots up a ton.

Reply:
I definitely understand your reasoning. Some of the bottom bench players have high PERs, and we don't want to over value them. I had not thought about this case when making edits, but it's also a case that doesn't happen often. Do you think a good solution would just to add another if/else if here. Such as:

if(played a lot for 2 seasons)
else if(played some for 2 season)
else (doesn't play, but has 2 years experience)

Reply:
I thought a good solution is what I have before, which was weighting the contribution of PER by minutes played, rather than a hard cutoff in minutes :)